lib/plugins/viewdb_timestamp_plugin.js

```diff
@@ -0,0 +1,22 @@
+var ViewDBTimestampPlugin = function (viewDb) {
+  var oldCollection = viewDb.collection;
+  viewDb.collection = function () {
+    var coll = oldCollection.apply(this, arguments);
+    if (!coll.__plugins_timestamp) {
+      coll.__plugins_timestamp = true;
+      var oldSave = coll.save;
+      coll.save = function (doc) {
+        doc.lastModified = new Date().valueOf();
+        oldSave.apply(this, arguments);
+      };
+      var oldInsert = coll.insert;
+      coll.insert = function (doc) {
+        doc.lastModified = new Date().valueOf();
```

Comment:
funderar på om vi skall köra med våra gamla vanliga namn på surikat
createDateTime och 
changeDateTime. Isåfall sätter vi create och change på insert och uppdaterar change på save.